### PR TITLE
Peddler's Daugther (80019)(Ready)

### PR DESCRIPTION
**Pull request by PummelsTL**
-Initial encounter up to choices
-Last Encounter

### CODE REVIEW

File: CSV/Card/80019.txt [hunk 1]
```diff
@@ -22,57 +22,57 @@ default
...
+        説明,I noticed a small girl sitting quietly and staring at me at the side of the arena.
+        説明,When I returned her gaze out of curiousity, she beckoned me over. I decided to go and see what she has to say.
         話者,自分,true,1
-        セリフ,ねーねー、おにいさんってすごく強いマスターさんだよね。\nおにいさんにちょっと聞いてほしいことがあるの。
-        セリフ,あのね、あのね。わたし最近体がすごく熱くなるんだ。\nこれって病気なのかな？ほら、さわってみて・・・？
-        説明,そう言って目の前の女の子は僕に手を握らせてきた。確かに普通の子供の手よりもなんだか温かいような気がする。
-        説明,だがそれだけではないような、なにか触っているだけで心地よくなるような不思議な感じがする。なんだろう、きもちいい・・・❤
+        セリフ,Hey, hey. Oniisan, you're a really strong cardmaster, right?\nThere's something I want to ask from you.
+        セリフ,You see, I've been feeling really hot recently.\nCould this be some sort of sickness? Just try how hot my body feels...
+        説明,After saying that, the girl grabbed my hand into her own. It certainly felt like her hand was warmer normal.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:30]: Warmer than normal would sound better here.

File: CSV/Card/80019.txt [hunk 2]
```diff
@@ -22,57 +22,57 @@ default
...
+        説明,However, that wasn't everything I felt. Just by holding her hand, I was feeling this strange but comfortable sensation. What could this be... It feels good❤
         フラッシュ,ピンク,
         話者,相手,
-        セリフ,（ううぅ❤なんで、体に力が入らない・・・❤）
+        セリフ,（Uuuh❤ Why... I'm feeling weak for some reason❤）
```
Comment (Fyurios) [inline CSV/Card/80019.txt:34]: Why... am I feeling weak        The for some reason here is implied with the why, I wanna say it isn't needed.

File: CSV/Card/80019.txt [hunk 3]
```diff
@@ -22,57 +22,57 @@ default
...
+        セリフ,My father is the same. Whenever I touch him,\n he can't help making a really pleasant-looking face and blushing❤
+        セリフ,He's a merchant,\n but recently he has always been staring at his cards that have pictures of girls.\nI wanted him to pay more attention to me, so I took those cards away from him.
+        セリフ,After that, I've been feeling that my body is really hot,\n and whenever I've touched a man's body,\nthey've become crazy for me❤
+        セリフ,Even my father has started to only think about me❤\nI tried holding onto his hand while staring at his face,\n and then...
+        セリフ,My father who has always been so cool and composed started to call my name again and again with an adorable face. I found it really fun how he trembled and moaned❤
+        セリフ,Oniisan, you're the same❤\nYou're falling into a daze even when I'm only holding your hand❤\nDoes it make you happy to hold hands with me?
+        説明,The girl adorably tilted her head when she asked that. I couldn't think straight when my mind was in a haze, so I ended up nodding.
+        セリフ,Really?\nAhaha❤ This is amazing❤ Even cardmasters can't falling for my charm❤
```
Comment (Fyurios) [inline CSV/Card/80019.txt:44]: Even cardmasters can't falling for my charm    Sounds a bit off. I'd say
Even card masters fall for my charms

File: CSV/Card/80019.txt [hunk 4]
```diff
@@ -22,57 +22,57 @@ default
...
+        セリフ,Hey, Oniisan. Why don't we go over to that corner?\nThere aren't many people there, so we can play by just the two of us❤
```
Comment (Fyurios) [inline CSV/Card/80019.txt:45]: play by   - play with

File: CSV/Card/80019.txt [hunk 5]
```diff
@@ -22,57 +22,57 @@ default
...
+        セリフ,You don't want to let go of my hand anymore, right?\nHere, I'll pull you over there❤
+        説明,The girl started to gently pull me over to a blindspot at the corner of the room. In that situation, I...
```
Comment (Fyurios) [inline CSV/Card/80019.txt:47]: blind spot is normally 2 words instead of one.

File: CSV/Card/80019.txt [hunk 6]
```diff
@@ -22,57 +22,57 @@ default
...
         {
             SE,&&振り払いSE,
             話者,自分,true,0.5
-            セリフ,うわわっ。えー、わたしと一緒に遊びたくないの？\nあんなに気持ちよさそうな顔してたのに、\n手を離しちゃうなんて・・・。
-            セリフ,うう、お父さんだけにしか、この魔法って効果がないのかな？\nお兄さんと「イイこと」して、\nあそびたかったなー・・・。ざんねん。
-            説明,そう言いながら女の子は自分の両手を組んで、にぎにぎと見せつけてくる。さっきまで感じていた彼女の手の平の気持ちよさが思い出されて、ついまた手を握りたくなってしまう。
-            説明,しかし僕は気を強く持って女の子から離れて受付の方に戻った。ただの小さい女の子のはずだがなぜだか危険な感じがする。これ以上近づかない方がいいだろう。
+            セリフ,Uah! Eeh? Don't you want to play with me?\nTo think that you would pull back your hand even when you made that pleasant face...
+            セリフ,Uuuh... Maybe this magic doesn't work on anyone else but my father?\nOniisan, I wanted to play and do "fun things" with you... But it's regrettable.
+            説明,While saying that, the girl showily rubbed her hands together in front of you. Seeing that, you couldn't help recalling how soft and amazing her hands felt just now, which made you want to hold them again.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:55]: the girl showily rubbed her hands together 

I want to say showily doesn't fit there due to grammar rules. Would probably be best to remove it.

File: CSV/Card/80019.txt [hunk 7]
```diff
@@ -22,57 +22,57 @@ default
...
+            説明,However, I held my ground and pulled back from the girl back to the counter. Even when she was just a small girl, you felt a sense of danger for some reason and felt like you shouldn't approach her any further.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:56]: You changed perspective here. You probably should make it first person like the rest of the descriptions.

Even when she    when doesn't fit here.

Even though she

File: CSV/Card/80019.txt [hunk 8]
```diff
@@ -22,57 +22,57 @@ default
...
     }else{
         話者,自分,true,1
-        セリフ,えへへ、やっぱり遊びたくなったの？\nもじもじしながらわたしのところにやってきて、\nおにいさんかわいいんだね❤
-        セリフ,ほら、さびしそうにしてるおててを、\nぎゅって握ってあげる❤\nきもちいいのでいっぱいになってねー❤
-        説明,そう言って女の子は僕の手を取り、片手をぴとっとふれさせてきた。
-        説明,触れるだけで手の平がぽわっとあたたかくなり、心地よい快感が流れ込んでくる。
+        セリフ,Ehehe❤ Oniisan, do you want to play with me after all?\nYou looked adorable when you were fidgeting while approaching me❤
+        セリフ,Here❤ Your hand is looking lonely,\n so I'll hold on to it❤\n Enjoy how good it feels❤
+        説明,After saying that, the girl took my hand to touch it.
+        説明,Just by hands with her, your palm felt warm and you could feel a pleasant sensation flowing into you.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:65]: Just by hands
I think you forgot a word here
Just by holding hands

File: CSV/Card/80019.txt [hunk 9]
```diff
@@ -22,57 +22,57 @@ default
...
         フラッシュ,ピンク,
         話者,相手,
-        セリフ,（ううぅ❤体に力が入らない・・・❤きもちいい・・・❤）
-        説明,うっとりしている僕を見て、女の子はにこりと笑い、さらに恋人同士がするようにぎゅっと指をからませてきた。
-        説明,指と指の間に女の子の指がすべりこみ、にぎにぎと軽い圧力を加えてくる。たったそれだけでまるで心を直接もみほぐされているように、体中が脱力してしまい、僕は夢心地にされてしまった。
+        セリフ,（Uuh❤ I'm losing the strenght in my body...❤ It feels good...❤)
```
Comment (Fyurios) [inline CSV/Card/80019.txt:68]: strenght 

strength

File: CSV/Card/80019.txt [hunk 10]
```diff
@@ -22,57 +22,57 @@ default
...
+        説明,When the girl saw my dreamy state, she gave me a wide smile and, furthermore, intertwined fingers with me like a lover.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:69]: fingers

her fingers

File: CSV/Card/80019.txt [hunk 11]
```diff
@@ -22,57 +22,57 @@ default
...
+        説明,She slid her fingers between mine and gently squeezed. Just by doing that, she made my whole body relax as if directly massaging my mind and put me into a trance.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:70]: Just by doing that, she made my whole body relax as if directly massaging my mind and put me into a trance.

Grammar here seems off. Perhaps something more like

That was all it took to put me in a trance, my body relaxing as if she were directly massaging my mind.

**Review by Fyurios — CHANGES_REQUESTED** [review]
Looks good overall. As I mentioned in a dm, we can't use normal commas so feel free to replace them with 、    、     、    、
I posted a few comments about some grammar stuff if you wanted to make some quick changes.

**Comment by PummelsTL** [comment]
Keep this open until I've finished translating all the scenes in the file.

**Comment by PummelsTL** [comment]
The file is fully translated and ready for last review.

File: CSV/Card/80019.txt [hunk 12]
```diff
@@ -1,119 +1,119 @@
...
+    &カード名,Peddler's Daugther
     &カード画像,NPC19.png
     &立ち絵差分影,NPC19_shadow.png
     画像部位設定,&カード画像,face,0.4,0.6,0.8,0.9
     画像部位設定,&カード画像,bust,0.5,0.7,0.7,0.9
     画像部位設定,&カード画像,hip,0.5,0.7,0.3,0.4
-    &フレーバーテキスト,行商人の子供の女の子。お父さんのことが大好き。遠くの村に行商に行ってから父親が全然自分にかまってくれなくなり寂しい思いをしていた。ある日、父親が持つサキュバスカードを偶然手に取り、そこから徐々にサキュバス化が始まる。性知識はないが手を握っているとお父さんが自分に夢中になってくれることに気づき、手を握って弄ぶようになる。次第にサキュバスとしての飢えを感じるようになり強い魔力を求めて主人公に狙いを定めた。ただし計算づくで行動しているわけではなく、本能に引かれるままなんとなく「おいしそう」な感じがする主人公に声をかけただけである。
-
+    &フレーバーテキスト,Peddler's daugther who loves her father. She has been feeling lonely after her father suddenly stopped playing with her after peddling to a faraway village. This led to her taking away the succubus card that was taking her father's attention; which became a trigger to her gradual transformation to a succubus. While she doesn't possess sexual knowledge; she started noticing her father getting crazy about her whenever she held hands with him. As her transformation progressed further; she started feeling hunger for stronger mana and set her aim on the protagonist. However; she didn't call out to out with bad intention but because she instinctually sensed him as something "delicious".
```
Comment (Fyurios) [inline CSV/Card/80019.txt:9]: A peddler's daughter

File: CSV/Card/80019.txt [hunk 13]
```diff
@@ -1,119 +1,119 @@
...
 @上機嫌な女の子
     背景,&&闘技場受付背景,
     BGM,&&闘技場受付BGM
     if,$$手つなぎドレイン女の子,==,0
     {
         話者,自分,true,1
-        説明,闘技場のすみでちょこんと座っている女の子が僕の方をじーっと見ている。
-        説明,気になってこちらも見返すと、僕に向かって手招きをしてきた。僕は彼女に近づいてみることにした。
+        説明,I noticed a small girl sitting quietly and staring at me at the side of the arena.
+        説明,When I returned her gaze out of curiousity; she beckoned me over; so I decided to see what she has to say.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:26]: curiosity

File: CSV/Card/80019.txt [hunk 14]
```diff
@@ -350,29 +350,29 @@ default
         ウエイト,1
         攻撃力加算,-2000,
         $$手つなぎドレイン女の子,+=,1
-        説明,その後ぼくは夜になるまで彼女に「あそんで」もらった。ズボンはべたべたになったが彼女が手を当てるだけでその精液はきれいに吸い取られてしまった。
-        説明,体中が異様なほど重く感じる、ぼくはふらふらになりながらなんとか宿屋に戻ったのだった。
+        説明,After that; I "played" with the girl until the night. Even when I had dirtied my pants; she only needed to give them a touch with her hand to absorb and clean off all the semen from them.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:353]: until late at night would probably sound better.

File: CSV/Card/80019.txt [hunk 15]
```diff
@@ -292,30 +292,30 @@ default
...
```
Comment (Fyurios) [inline CSV/Card/80019.txt:295]: until late at night would probably sound better.

File: CSV/Card/80019.txt [hunk 16]
```diff
@@ -224,54 +224,54 @@ default
         ウエイト,1
         攻撃力加算,-1000,
         $$手つなぎドレイン女の子,+=,1
-        説明,その後ぼくは夜になるまで彼女に「あそんで」もらった。ズボンはべたべたになったが彼女が手を当てるだけでその精液はきれいに吸い取られてしまった。
-        説明,体中が異様なほど重く感じる、ぼくはふらふらになりながらなんとか宿屋に戻ったのだった。
+        説明,After that; I "played" with the girl until the night. Even when I had dirtied my pants; she only needed to give them a touch with her hand to absorb and clean off all the semen from them.
```
Comment (Fyurios) [inline CSV/Card/80019.txt:227]: until late at night would probably sound better.

**Review by Fyurios — CHANGES_REQUESTED** [review]
Just saw a few grammar things off, mostly the same line in different sections.

**Comment by PummelsTL** [comment]
Added the grammar fixes.

**Pull request merged**